data/json/terrain.json

```diff
@@ -6837,5 +6837,225 @@
+      ]
+    },
+    "bash": {
+      "str_min": 12,
+      "str_max": 80,
+      "sound": "clang!",
+      "sound_fail": "ting.",
+      "ter_set": "t_floor",
+      "items": [
+        { "item": "pipe", "amount": 3, "minamount": 1 },
+        { "item": "chain", "amount": 1, "chance": 10 },
+        { "item": "scrap", "amount": 5, "minamount": 1 }
+      ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_machinery_light",
+    "name": "light machinery",
+    "symbol": "$",
+    "color": "dkgray",
+    "move_cost": 10,
+    "flags": [ "TRANSPARENT", "BASHABLE", "CONTAINER", "FLAMMABLE", "PLACE_ITEM" ],
+    "deconstruct": {
+      "ter_set": "t_floor",
+      "items": [
+        { "item": "wire", "amount": 3, "minamount": 1 },
+        { "item": "pipe", "amount": 2, "minamount": 1 },
+        { "item": "chain", "amount": 1, "chance": 40 },
+        { "item": "cu_pipe", "amount": 1, "chance": 40 },
+        { "item": "scrap", "amount": 4, "minamount": 1 },
+        { "item": "hose", "amount": 1 },
+        { "item": "steel_chunk", "amount": 5, "minamount": 1 },
+        { "item": "bearing", "amount": 12, "minamount": 4 },
+        { "item": "frame", "amount": 1, "chance": 50 },
+        { "item": "motor", "amount": 1, "chance": 50 }
+      ]
+    },
+    "bash": {
+      "str_min": 16,
+      "str_max": 80,
+      "sound": "clang!",
+      "sound_fail": "ting.",
+      "ter_set": "t_floor",
+      "items": [
+        { "item": "wire", "amount": 1 },
+        { "item": "pipe", "amount": 2, "chance": 40 },
+        { "item": "chain", "amount": 1, "chance": 20 },
+        { "item": "cu_pipe", "amount": 1, "chance": 10 },
+        { "item": "scrap", "amount": 8, "minamount": 3 },
+        { "item": "steel_chunk", "amount": 4, "minamount": 1 },
+        { "item": "bearing", "amount": 8, "minamount": 2 },
+        { "item": "frame", "amount": 1, "chance": 20 },
+        { "item": "motor", "amount": 1, "chance": 10 }
+      ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_machinery_heavy",
+    "name": "heavy machinery",
+    "symbol": "%",
+    "color": "ltgray",
+    "move_cost": 0,
+    "flags": [ "BASHABLE", "CONTAINER", "SEALED", "PLACE_ITEM" ],
+    "deconstruct": {
+      "ter_set": "t_floor",
+      "items": [
+        { "item": "wire", "amount": 3, "minamount": 1 },
+        { "item": "pipe", "amount": 2, "minamount": 1 },
+        { "item": "chain", "amount": 1, "chance": 60 },
+        { "item": "cu_pipe", "amount": 1, "chance": 20 },
+        { "item": "steel_lump", "amount": 2, "minamount": 1 },
+        { "item": "hose", "amount": 1 },
+        { "item": "sheet_metal", "amount": 3, "minamount": 1 },
+        { "item": "steel_chunk", "amount": 3, "minamount": 1 },
+        { "item": "bearing", "amount": 12, "minamount": 4 },
+        { "item": "frame", "amount": 1, "chance": 60 },
+        { "item": "motor", "amount": 1, "chance": 30 },
+        { "item": "metal_tank_small", "amount": 1, "chance": 30 },
```

Comment:
Apparently this item is an issue for Travis:
`src/item_group.cpp:116 [virtual void Single_item_creator::check_consistency() const] item id metal_tank_small is unknown`

`src/item_group.cpp:116 [virtual void Single_item_creator::check_consistency() const] item id metal_tank_small is unknown`

-- However, it's listed in migration.json (Line 333):
```json
  {
    "id": "metal_tank_small",
    "type": "MIGRATION",
    "replace": "jerrycan"
  },
```

Replacing with jerrycan for the simple fix, but it shouldn't be throwing an error, right?